src/partition/repart.c

```diff
@@ -1699,6 +1702,59 @@ static int context_dump_partitions(Context *context, const char *node) {
         return 0;
 }
 
+static int generate_report(Context *context, const char *node) {
+        _cleanup_(table_unrefp) Table *t = NULL;
+
+        Partition *p;
+        int r;
+
+        if (!arg_json)
+                return 0;
+
+        t = table_new("activity", "type", "label", "uuid", "file", "node", "offset", "old_size", "new_size", "old_padding", "new_padding");
+        if (!t)
+                return log_oom();
+
+        LIST_FOREACH(partitions, p, context->partitions) {
+                _cleanup_free_ char *partname = NULL;
+                char uuid_buffer[ID128_UUID_STRING_MAX];
+                const char *label = NULL;
+                const char *activity = NULL;;
+
+                if (p->dropped)
+                        activity = "dropped";
```

Comment:
the dropped ones should probably never be shown

Reply:
Why? It was listed *somewhere*, so it should IMHO show up in the report:-)

Reply:
I removed the dropped partitions now.